oak/server/rust/oak_glue/src/lib.rs

```diff
@@ -101,7 +101,7 @@ pub unsafe extern "C" fn glue_start(
             }
         };
         let runtime_config = oak_runtime::RuntimeConfiguration {
-            metrics_port: Some(3030),
+            metrics_port: Some(9090),
```

Comment:
Why the port has changed?

Reply:
It is the default port that k8s service discovery (indicated by `kubernetes_sd_configs`) looks for. I guess I could change the configuration for the prometheus server, but I thought it would be better to keep the standard default. The prometheus server itself runs on port 9090 too. That is why earlier I thought we have to use a different port for our metrics server, but that is not necessary.